Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNames.proto

```diff
@@ -11,75 +11,101 @@ import "google/api/field_behavior.proto";
 service ResourceNames {
   option (google.api.default_host) = "resourcenames.example.com";
 
-  // Test top-level resource definition.
+  // Test simple resource definition.
   rpc SimpleMethod(SimpleResource) returns(Response) {
     option (google.api.method_signature) = "name";
   }
 
-  // Test resource definition inlined in the proto message.
-  rpc SimpleInlineMethod(SimpleInlineResource) returns(Response) {
+  // Test simple resource reference.
+  rpc SimpleMethodRef(SimpleResourceRef) returns(Response) {
+    option (google.api.method_signature) = "simple_resource";
+  }
+
+  // Test multi resource definition.
+  rpc MultiMethod(MultiResource) returns(Response) {
     option (google.api.method_signature) = "name";
   }
 
-  // Test repeated resource.
-  rpc SimpleRepeatedMethod(SimpleRepeatedResource) returns(Response) {
-    option (google.api.method_signature) = "names";
+  // Test multi resource reference.
+  rpc MultiMethodRef(MultiResourceRef) returns(Response) {
+    option (google.api.method_signature) = "multi_resource";
   }
 
-  // Test wilcard resource.
-  rpc WildcardMethod(WildcardResource) returns(Response) {
-    option (google.api.method_signature) = "name,names,required_single,required_repeated";
+  // Test future multi resource (FUTURE_MULTI_PATTERN).
+  rpc FutureMultiMethod(FutureMultiResource) returns(Response) {
```

Comment:
I can't see any methods being generated for this. Possibly I haven't opened a diff? Or is it actually missing?

Reply:
Well spotted, thank you :)
They were be generated, but were not present in the test file, which doesn't contain the entire generated file.
Done.